### PR TITLE
Add simple menu on datepicker

### PR DESCRIPTION
**Pull request by HichamELBSI**
Signed-off-by: HichamELBSI <elabbassih@gmail.com>

# DatePicker

## Description
Add month and year menu on DatePicker

## Demo
Example on : [deployed story link]

## API
No additional API

## Voiceover
datesr

### CODE REVIEW

File: packages/strapi-design-system/src/DatePicker/DatePicker.js [hunk 1]
```diff
@@ -68,49 +96,69 @@ export const DatePicker = ({ initialDate, selectedDate, onChange, label, selecte
...
+                <Box paddingBottom={4}>
+                  <Row>
+                    <SimpleMenu id="year" label={months[date.getMonth()]}>
+                      {months.map((month) => (
+                        <MenuItem key={month} onClick={() => handleMonthChange(month)}>
+                          {month}
+                        </MenuItem>
+                      ))}
+                    </SimpleMenu>
+                    <SimpleMenu id="months" label={date.getFullYear()}>
```
Comment (mfrachet) [inline packages/strapi-design-system/src/DatePicker/DatePicker.js:109]: There's a proptypes error on this I think (number and string)

**Review by mfrachet — APPROVED** [review]
Expect the PropTypes error, very nice job 😊

**Comment by mfrachet** [comment]
Also, I've noted that when selecting a date and opening the picker again, it does select the current year and not the previously selected one 🤔 . is that wanted?

**Comment by HichamELBSI** [comment]
> Also, I've noted that when selecting a date and opening the picker again, it does select the current year and not the previously selected one 🤔 . is that wanted?

Not sure I understand. When you select a date, the displayed month and year should be those of the selected date.

**Comment by mfrachet** [comment]
Yes, but when I open the menu again, the date of the current year (2021) is highlighted in blue, even if the year in the menu is "2014"

File: packages/strapi-design-system/src/DatePicker/DatePicker.js (hunk 1 above)
Comment (HichamELBSI) [reply packages/strapi-design-system/src/DatePicker/DatePicker.js:109]: Nice catch ! thanks

**Pull request merged**